lib/GhcideSteal.hs

```diff
@@ -42,13 +39,13 @@ realSrcSpanToRange real =
 
 realSrcLocToPosition :: RealSrcLoc -> Position
 realSrcLocToPosition real =
-  Position (srcLocLine real - 1) (srcLocCol real - 1)
+  Position (fromIntegral $ srcLocLine real - 1) (fromIntegral $ srcLocCol real - 1)
```

Comment:
I added a bunch of `fromIntegral :: Int -> UInt`, I should probably create a safe conversion function here